src/SDKs/PostgreSQL/Management.PostgreSQL/generate.ps1

```diff
@@ -0,0 +1 @@
+powershell.exe -ExecutionPolicy Bypass -NoLogo -NonInteractive -NoProfile -File "$(split-path $SCRIPT:MyInvocation.MyCommand.Path -parent)\..\..\..\..\tools\generateTool.ps1" -ResourceProvider "postgresql/resource-manager" -PowershellInvoker  -AutoRestVersion "latest"
```

Comment:
@rohit-joy if you use generate.ps1 it will create .txt file with all the meta data.

Reply:
Done